process/subgroups.md

```diff
@@ -0,0 +1,69 @@
+# Subgroups
+
+Subgroups of the CG are created when one (or both) of the following apply:
+
+- An area of interest that needs a lot of discussion. Creating a subgroup with regular meetings provides more bandwidth for this discussion.
+- An area of interest is only relevant to a small group of participants. In these cases, subgroup meetings give a more focused venue. This way, participants don’t have concerns about wasting the CG’s time with something that the rest of the CG is not engaged in.
+
+To create a subgroup, you should add a poll to a CG meeting agenda with at least 7 days notice. As part of the poll, you should provide information about who will be chairing the subgroup.
+
+## Current Subgroups
+
+| Subgroup  | Chair(s)  | Discussion Repo  | Meetings  |
+|---|---|---|---|
+| Debugging  | Derek Schuff (@dschuff)  | [Repo](https://github.com/WebAssembly/debugging)  | no meetings currently scheduled, but discussion or agenda suggestions are welcome |
+| GC  | Thomas Lively (@tlively)  | [Repo](https://github.com/WebAssembly/gc)  | [Agendas](https://github.com/WebAssembly/gc/issues?q=is%3Aissue+%22Agenda+for+subgroup+meeting%22)  |
+| SIMD  | Petr Penzin (@penzn), Ng Zhi An (@ngzhian)  | ? | [Agendas](https://github.com/WebAssembly/meetings/tree/main/simd/)  |
```

Comment:
https://github.com/WebAssembly/simd ?

Reply:
The SIMD repo has been archived after being merged into the spec, either one of the newer SIMD proposal issue trackers should work because they're mirrored for agenda issues. 
https://github.com/WebAssembly/relaxed-simd/issues
https://github.com/WebAssembly/flexible-vectors/issues

Reply:
That is accurate, thank you. We cold switch to tracking in the meetings repo, if that is easier to follow, but keep announcements in the proposals.